package.json

```diff
@@ -114,6 +114,7 @@
     "marked-extended-tables": "^1.1.0",
     "marked-gfm-heading-id": "^4.0.1",
     "marked-smartypants-lite": "^1.0.3",
+    "marked-subsuper-text": "^1.0.1",
```

Comment:
Is this package live? I can't see it in NPM.

Reply:
Should be? I can see it and pull it.

https://www.npmjs.com/package/marked-subsuper-text ?

Reply:
Weird. It doesn't show up when I search for it. But I can open it. Will review over there as well as part of this PR.